deployment/jenkins/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM jenkins/jenkins:2.190.1
+
+USER root
```

Comment:
Whoa we need root here? :o

Reply:
Yeah any apt-get commands need to run as sudo, so root is needed :(

Reply:
Gross!